### ISSUE TITLE
Guys, It's over.

### ISSUE THREAD

**Issue by WolfKetchum** [finndev/PokeBuddy#981]
Niantic is now perma banning almost everyone who is using bots or using fake GPS spoofing, until pokebuddy updates to be immune against the perma banning, it's over.

**Comment by onilax** [comment]
are you banned already?

**Comment by JavierBarrrera** [comment]
Are you sure? D:

**Comment by trialfail** [comment]
can the creator of pokebuddy make it so we cant get found??

**Comment by NegroSorete** [comment]
Yep GG guys. I lose mi acc lvl 28, but i dont care. Uninstalling right now

**Comment by PSUolive** [comment]
No one on the internet has proven for a FACT that they are banning anyone.. 
Dont spread the same rumors unless you can back up your claim with undeniable proof that they are banning anyone. And at this point there is no proof what so ever that they are banning anyone. Also ALOT of fake HOAX pictures going around  about the bans.

**Comment by LeoAchilles** [comment]
still botting right now but haven't faced any ban yet (soft ban, ip ban, perm ban)

**Comment by trialfail** [comment]
@LeoAchilles what is your runtime and are you using a vpn?

**Comment by JavierBarrrera** [comment]
@PSUolive Ye, that make sense.

**Comment by onilax** [comment]
try to reduce the bot speed like human walking speed.. we should consider the possibility on how they figure it out. maybe in the speed, teleportation, IP.. and other things.. maybe we could consider the delay in using every functions like catching pokemon with delay, using item with delay, every action maybe we could put a delay so they wont consider it as bot.

**Comment by onilax** [comment]
i tried yesterday 30km/hr. after 5-10min. i got ip ban... but then today, i put 15km/hr. 2hr and 30min.. still running... as of now.. =D

**Comment by PSUolive** [comment]
Anyone with a brain has been calling it "Server Issues" 
http://ispokemongodownornot.com/ also shows US Google Login servers down or having issues. 

That Data is backed up by other Server Status sites such as 
http://www.mmoserverstatus.com/pokemongo

That also shows US server and google Login problems.

**Comment by LeoAchilles** [comment]
@trialfail  
- "Farm Pokestops", "Catch Pokemon", Transfer bad Pokemon", "Use Incubators".
- Walk speed 7km/hr
- bot nearby my home
- did not use any VPN
- botting one account currently

**Comment by w282619307** [comment]
cant cath pokemon now

**Comment by Skolzack** [comment]
@PSUolive Of course it's not a server problem. I got "hardbanned" more than 10hours and still, I can catch pokemons, hatch eggs, transfer etc but can't spin pokestops. Others are facing catchflee and softban.. Niantic did really a massive work to stop with the botters. Maybe they implemented some new function to detect any botter service and automatically ban it, such as speed or something..

**Comment by w282619307** [comment]
[P - 12:45:39] [Catch Pokemon] Magikarp got away using a ItemPokeBall, status: CatchFlee

**Comment by PSUolive** [comment]
@Skolzack  You got "HardBanned" but you can still play? Then you are not ban!

**Comment by JavierBarrrera** [comment]
An option to vary the speed of 15 km / hr - 20 km / hr, maybe.

**Comment by Skolzack** [comment]
@PSUolive It's a new type of ban.. not softban anymore cuz softban don't lasts more than 5hours.

**Comment by psblackz** [comment]
Im banner since yesterdar in same ACC , i think its Perm ban.. GG

**Comment by Skolzack** [comment]
@psblackz With permaban you can't enter on your account, you can or not?

**Comment by PSUolive** [comment]
Multiple Websites show Login Server problems. 
![server status - ispokemongodownornot](https://cloud.githubusercontent.com/assets/20951319/17630031/92bef7f6-608b-11e6-9696-d839da1c69a6.JPG)
![server status - mmoserverstatus](https://cloud.githubusercontent.com/assets/20951319/17630034/943eba8a-608b-11e6-85d3-f017a21ec030.JPG)

**Comment by psblackz** [comment]
I canr spin anything and catch anythong since yesterday 14h already and nothong

**Comment by Skolzack** [comment]
@PSUolive What login problems has to do with the new "hardban" thing? lol

**Comment by NegroSorete** [comment]
Permanent ban https://k61.kn3.net/4B30C435D.jpg

**Comment by Skolzack** [comment]
@NegroSorete It's not a Permaban dude... stop saying that

**Comment by NegroSorete** [comment]
@Skolzack It's ban dude, i cant login from yesterday... but i can still login with my other acc. So GG

**Comment by PSUolive** [comment]
@NegroSorete  what country are you from? Do you login with Google or PTC?

**Comment by JavierBarrrera** [comment]
You might get this from Niantic https://support.pokemongo.nianticlabs.com/hc/en-us/articles/225146308

And you might see one of these screens http://imgur.com/a/hvNvT

There are a large number of bans of this type occurring, which have been reported in some of the bot slack channels. While we're not 100% sure they're permanent yet, better safe than sorry.
To clear up confusion there are several types of bans:

**Soft IP ban Ban the IP but not the account (not permanent).** All accounts on the ip will be unable to see pokestops and pokemon. Usually goes away after 20 mins of not accessing the pogo api from that IP.
**Hard IP ban Ban the IP perm.** This happened to cloud providers like AWS. You can't log in or access pogo at all from one of these IPs. Never goes away.
Soft ban Account specific - there are now multiple types of softbans: (time to go away varies)
Spinning pokestops gives nothing, and pokemon always run on the first ball. (this was the original softban)
Spinning pokestops is fine, but pokemon always run. (added today)
All pokestops say out of range, and no pokemon ever spawn (added recently) ... there may be more
Hard ban Perma ban (Read data error on phone). This is the new ban that botters have been getting. If you get a login error, it does not mean you are banned for sure, but if you were botting or teleporting and you can't log in on that account from any IP, and you can use other accounts - good chance thats a hardban.

EDIT: People wondering what actions cause the ban. It is still unknown, but it seems related to highly frequent requests involving teleporting. However, any third party api usage or GPS spoofing voilates their TOS and it is possible you can receive a ban. I recommend not using any 3rd party service or spoofing for at least 24 hours. IMO, If you have never botted or spoofed/sniped, it is unlikely you will get banned. But I would avoid using any services that require your login until this is sorted out.

**Comment by psblackz** [comment]
I think i am, i cant play even on my phone since yesterday

**Comment by AHNMA** [comment]
PSUolive you dont have any facts... there are A LOT people since yesterday that cant login... like me! Otherwise my 2nd smurfaccount is working well... how in the world goes that confirm with your theory?

**Comment by Skolzack** [comment]
@JavierBarrrera Fake screenshot.

**Comment by inkompetent** [comment]
#984

**Comment by JavierBarrrera** [comment]
@Skolzack Ye, maybe. :/

**Comment by NegroSorete** [comment]
@JavierBarrrera Fake screenshot? just keep boting dude lul. GL

**Comment by PSUolive** [comment]
@AHNMA there is thousands of reports of not being able to login from Botters and from those who play legit..  Look at the Current Server Status's. Alot have been down and offline for over 13 hours now

http://www.mmoserverstatus.com/pokemon_go

**Comment by Skolzack** [comment]
@PSUolive Where's the reports of legit players that can't are able to login?

**Comment by JavierBarrrera** [comment]
Just waiting.

**Comment by PSUolive** [comment]
The servers show as offline for 13 hours (12:00 EST)... Kinda weird that the Bot stopped 13 hours ago.at 12:30 EST 
![bot stopped working when server went down](https://cloud.githubusercontent.com/assets/20951319/17630745/9937374e-608e-11e6-9c08-05784b2f674d.png)

**Comment by Skolzack** [comment]
@PSUolive I don't get these errors. Maybe it's a real problem server but not for all

**Comment by zumisogetsu** [comment]
@NegroSorete The screenshot doesn't say anything except that the servers were busy. LOL, nice try spreading rumors.

**Comment by DavideRaggi** [comment]
So.. it's a  bot problem that need an update for bypassing the new PGo server protections or what ?

**Comment by Skolzack** [comment]
@DavideRaggi Maybe Niantic patched the 45spin bypass and improved the time of the softbans..

**Comment by DavideRaggi** [comment]
Or maybe Niantic has patched this bot forever... I don't think so, but it's possible

**Comment by jacearchitect69** [comment]
![pokeban](https://cloud.githubusercontent.com/assets/20736839/17631194/55fad62c-60c3-11e6-90d4-23c69aa01649.jpg)

**Comment by jacearchitect69** [comment]
this is my perma ban since this morning after i trid hotspot vpn
i guess too is perma ban.... by bye to my level 35^^

**Comment by AlexCtasy** [comment]
What If i get that softban that all is working fine, u can catch pokemon, see pokestops all is ok, but when u go spin some pokestops says "try again later".. I need wait some hours to get unbanned or maybe it's time to say bb account?

**Comment by MJH94** [comment]
Same problem here with pokestops saying try again later

**Comment by MariaBitterlich** [comment]
Bot was running 2 hour straight without any problems. I stopped it now, because of all the rumours and will wait till the dust settles.

**Comment by MJH94** [comment]
MY bot is running fine with my secondary account. im even running it on 100km/h. for about 2 hours now no problems

**Comment by jacearchitect69** [comment]
[P - 20:08:51] Retrying request https://pgorelease.nianticlabs.com/plfe/368/rpc

what is this? with other account i am trying to see what call for the soft ban...
after 4 minutes i got this message then softban...

**Comment by AlexCtasy** [comment]
Cant see ur img :S

**Comment by Artillery101** [comment]
So what does mean getting the message that was screenshot above?
